auth/encryptcli/src/main/java/com/seagates3/authpassencryptcli/AuthEncryptCLI.java

```diff
@@ -114,18 +114,19 @@ public static String parseArgs(String[] args) {
      * @throws GeneralSecurityException
      * @throws Exception
      */
-    public static String processEncryptRequest(String passwd)
-                       throws GeneralSecurityException, Exception {
+   public
+    static String processEncryptRequest(String passwd)
+        throws GeneralSecurityException,
+        Exception {
 
-        String encryptedPasswd = null;
-        if(passwd == null || passwd.isEmpty()
-                          || passwd.matches(".*([ \t]).*")) {
-            logger.error("Password is null or empty or contains spaces");
-            System.err.println("Invalid Password Value.");
-            throw new Exception("Invalid Password Value.");
-        }
+      String encryptedPasswd = null;
+      if (passwd == null || passwd.isEmpty() || passwd.matches(".*([ \t]).*")) {
+        logger.error("Password is null or empty or contains spaces");
+        System.err.println("Invalid Password Value.");
```

Comment:
I guess yes, as this jar can also be run from shell or console, the user should get error or output on console.

Also I have not done any code changes to this file, just fixed "clang-format" issues as I was getting build failure because of this.

Reply:
okay.. pls remove this print statement.

Reply:
Sachitanand, I guess we should keep this... Suppose the user runs the jar on the cmd-line, and this error occurs. If we remove the println statement, then the user will not get to know why the CLIfailed !! They have to check the logs...

So better to keep this !!!

Reply:
yes. sounds good.